src/main.py

```diff
@@ -142,6 +149,22 @@ def do_activate(self) -> None:  # pylint: disable=arguments-differ
 
         self.win.present()
 
+    def check_desktop_terminals(self) -> None:
+        """Look for an installed terminal for desktop entries"""
+        terminals = ("xdg-terminal-exec", "kgx", "gnome-terminal", "konsole", "xterm")
```

Comment:
The values probably should be read from the enum in `data/hu.kramo.Cartridges.gschema.xml.in`.  
We must limit repetition of values to make the code more readable and expandable.

Reply:
Nicknames aren't readable.